lib/assets/javascripts/dmproadmap/forms.js

```diff
@@ -1,39 +1,56 @@
 // ---------------------------------------------------------------------------
+function remoteSave(target, method, data){
+  $("div.roadmap-info-box span:not(.aria-only)").parent().css('visibility', 'hidden').fadeOut()
+
+  // Update the visbility when the user clicks on the radio button
+  $.ajax({
+    url: target, 
+    type: method,
+    data: data,
+    contentType: 'application/json',
+    accepts: 'application/json'
+  }).done(function(data){
+    $("#notification-area span:not(.aria-only)").html(data['msg']).css('width', 'auto')
+          .attr('role', (data['code'] === 1 ? 'status' : 'alert'))
+          .attr('class', (data['code'] === 1 ? 'roadmap-info-box' : 'roadmap-alert-box'))
+          .parent().css('visibility', 'visible').fadeIn();
+  });
```

Comment:
I encourage you to use .then(resolved,rejected) rather than just done, otherwise for any error the user will not be informed at all.